src/views/Send/Send.vue

```diff
@@ -378,7 +391,10 @@ export default {
       return unit
     },
     isValidAddress() {
-      return chains[cryptoassets[this.asset].chain].isValidAddress(this.address, this.activeNetwork)
+      return (
+        this.isValidDomain ||
+        chains[cryptoassets[this.asset].chain].isValidAddress(this.address, this.activeNetwork)
+      )
     },
     addressError() {
```

Comment:
When I enter a domain that can't be resolved (doesn't exist), I get error:
`'Wrong format. Please check the address.'`
But in this case it should say `Name not found` or something along those lines

Reply:
For this, we need to call isValidTld function to check if this is a domain or gibberish. Which is an async function and would add unwanted complexity